testbenches/Makefile

```diff
@@ -79,7 +79,7 @@ $(TARGETS): $(SIMLIBS)
 		DEBUG=$(DEBUG) TURBO=$(TURBO)
 
 clean: $(addsuffix .clean,$(TARGETS)) simlibs.clean
-	rm -rf *.log *.jou
+	rm -rf *.log *.jou *.csv
```

Comment:
Let's just remove vanilla_stats.csv (or whatever the filenames are) Users might have other .csv files and I don't want  to destroy them